### DISCUSSION TITLE
Custom `output` path can only work if it's in `node_modules`

### DISCUSSION THREAD

**Discussion by shelleypham** [prisma/prisma#23896] [Q&A]
### Bug description

I followed the Prisma documentation for custom output:
https://www.prisma.io/docs/orm/prisma-client/setup-and-configuration/generating-prisma-client#using-a-custom-output-path

My stack:
- nextjs 14
- pnpm
- docker
- postgres
- jest

### Problem

I have two databases, `database_A` and `database_B`, that I have are running on docker containers via `docker-compose`. My Prisma client is not containerized. Each have their own schemas `schema_A.prisma` and `schema_B.prisma`.

 Both schemas generated successfully, pushed into my postgresql, and was seeded with data. `schema_A.prisma` uses the default custom output, but `schema_B.prisma` uses a custom output. 

Everything that involves `schema_A.prisma` runs smoothly; however, whenever I am accessing the client from `schema_B.prisma` (the one with the custom output), I encounter this error:

```
    PrismaClient is unable to run in this browser environment, or has been bundled for the browser (running in `Node.js`).
    If this is unexpected, please open an issue: https://pris.ly/prisma-prisma-bug-report
```

`schema_A.prisma`:
```
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}
```

`schema_B.prisma`:
```
generator client {
  provider = "prisma-client-js"
  output   = "../tests/generated/client"
}

datasource db {
  provider = "postgresql"
  url      = env("TEST_DATABASE_URL")
}
```

### The workaround

As the output above `../tests/generated/client` does not work, I tried having it in `node_modules` and everything worked smoothly.

`schema_B.prisma`:
```
generator client {
  provider = "prisma-client-js"
  output   = "./node_modules/@internal-stuff/tests/prisma-two/client"
}

datasource db {
  provider = "postgresql"
  url      = env("TEST_DATABASE_URL")
}
```

### How to reproduce

`schema_A.prisma`:
```
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}
```

`schema_B.prisma`:
```
generator client {
  provider = "prisma-client-js"
  output   = "../tests/generated/client"
}

datasource db {
  provider = "postgresql"
  url      = env("TEST_DATABASE_URL")
}
```

### Change to

`schema_B.prisma`:
```
generator client {
  provider = "prisma-client-js"
  output   = "./node_modules/@internal-stuff/tests/prisma-two/client"
}

datasource db {
  provider = "postgresql"
  url      = env("TEST_DATABASE_URL")
}
```

### Expected behavior

I expect to reference the Prisma client with my custom output

### Prisma information

`schema_A.prisma`:
```
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}
```

`schema_B.prisma`:
```
generator client {
  provider = "prisma-client-js"
  output   = "../tests/generated/client"
}

datasource db {
  provider = "postgresql"
  url      = env("TEST_DATABASE_URL")
}
```

### Environment & setup

- OS:  macOS
- Database:  PostgreSQL
- Node.js version: v18.17.1

### Prisma Version

```
5.12.1
```

**Comment by ludralph** [comment]
Hi @shelleypham 👋 

> Everything that involves schema_A.prisma runs smoothly; however, whenever I am accessing the client from schema_B.prisma (the one with the custom output), I encounter this error:

    PrismaClient is unable to run in this browser environment, or has been bundled for the browser (running in `Node.js`).
    If this is unexpected, please open an issue: https://pris.ly/prisma-prisma-bug-report

The error above is not usually associated with using a custom output in Prisma but mostly using Prisma in middleware within Nextjs or calling Prisma on the client. Can you please share a minimal reproduction of the repository so we can take a look?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
